### PR TITLE
Add m8tro-bootstrap

### PR DESCRIPTION
**Pull request by idleberg**

### CODE REVIEW

File: files/m8tro-bootstrap/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "m8tro-bootstrap",
+  "repo": "idleberg/m8tro-bootstrap",
+  "files": {
+    "include": ["./dist/m8tro.css", "./dist/m8tro.min.css"]
```
Comment (jimaek) [inline files/m8tro-bootstrap/update.json]: Thats not correct, your settings will create a folder called "dist".
Do this instead:
```
"files": {
    "basePath": "dist/",
    "include": ["m8tro.min.css"]
...
```

Also do you really need the un-minified file? Maybe its best to only hos the min.css?

**Comment by idleberg** [comment]
To be honest, I keep wondering why Bootstrap or Bootswatch put their unminified CSS on CDNs. I don't mind getting only the minified version hosted.

**Comment by jimaek** [comment]
One more change, please rename the directory to `bootstrap.m8tro` after that squash your commits and its good to go.

Thanks

**Comment by idleberg** [comment]
Any specific reason why I should rename this? It's using the same name on Bower and npm, so I think it makes sense to keep it consistent.

**Comment by jimaek** [comment]
To follow the existing format
https://db.tt/mJ4KKaPW
https://db.tt/Q47Mgus5

{mainLib}.{pluginName}

**Comment by jimaek** [comment]
Nice, please squash the commits into 1 and then I can merge

**Pull request closed without merging**